### PR TITLE
DMP-2818 Minor tweak to empty handler logic

### PR DESCRIPTION
**Pull request by jackmaloney**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapper.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public EventHandlerEntity mapFromEventMappingAndMakeActive(EventMapping eventMap
     }
 
     private String getHandlerMappingOrNullHandler(String handler) {
-        return handler != null ? handler : DartsEventNullHandler.class.getSimpleName();
+        return handler != null && !handler.equalsIgnoreCase("") ? handler : DartsEventNullHandler.class.getSimpleName();
     }
```
Comment (mario-paniccia) [inline src/main/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapper.java:33]: how about we simplify using Guava `Strings.isNullOrEmpty()`?

**Review by mario-paniccia — APPROVED** [review]
approved with one suggestion if you want

File: src/main/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapper.java (hunk 1 above)
Comment (chrisbellingham-hmcts) [reply src/main/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapper.java:33]: I was going to make the same suggestion as @mario-paniccia , but would suggest `StringUtils.isNotBlank()` from `apache.commons` which covers null, empty and blank cases.

File: src/test/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapperTest.java [hunk 2]
```diff
@@ -43,11 +46,13 @@ void mapsEventMapperToEventHandlerEntityCorrectly() {
             .hasFieldOrPropertyWithValue("createdBy", USER_ACCOUNT_ENTITY);
     }
 
-    @Test
-    void mapsEventMapperToEventHandlerEntityCorrectlyWhenHandlerIsEmpty() {
+    @ParameterizedTest
+    @NullSource
+    @ValueSource(strings = {""})
```
Comment (chrisbellingham-hmcts) [inline src/test/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapperTest.java:51]: Perhaps also test for blank string e.g.:
```suggestion
    @ValueSource(strings = {"", " "})
```

**Pull request merged**